### PR TITLE
30 replace

### PR DESCRIPTION
**Pull request by levBagryansky**
Solving #30 
So now you can reference to the groups in the replacing text.

### CODE REVIEW

**Comment by levBagryansky** [comment]
@Graur please review

File: src/test/eo/org/eolang/txt/regex-tests.eo [hunk 1]
```diff
@@ -212,3 +221,33 @@
...
+      "Ga1Ga GA2GB\n"
+
+[] > group-ref-2
+  assert-that > @
+    replaced.
+      compile.
+        QQ.txt.regex "/([xyz]+)/"
+      "abxxxcd"
+      "$0"
+    $.equal-to "abxxxcd"
```
Comment (Graur) [inline src/test/eo/org/eolang/txt/regex-tests.eo:253]: @levBagryansky Can you add an extra test with Unicode string? Like `Привет, 世界` or something else

**Review by Graur — CHANGES_REQUESTED** [review]
@levBagryansky Good one, just one comment from my side

**Review by Graur — APPROVED** [review]
@levBagryansky Looks good to me, thanks!

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by levBagryansky** [comment]
@yegor256 please take a look

**Comment by yegor256** [comment]
@levBagryansky try to give more descriptive titles to your PRs

**Comment by yegor256** [comment]
@rultor merge
